README.md

```diff
@@ -12,172 +12,114 @@ This project provides the [JSON schemas](https://json-schema.org/) for defining
 
 ## Introduction
 
-Catalogues often contain diverse types of data, and consistent metadata structures are crucial for effective data management and searching. This JSON schema defines a standardised format for describing metadata fields, their types, and any constraints or requirements associated with each field.
+The National Archives (TNA) collects and preserves digital records from various sources, including government departments, courts and public enquiries. To manage these records effectively, TNA requires a consistent and structured approach to metadata creation and management.  
+While JSON is probably the most popular format for exchanging data, JSON Schema is the vocabulary that enables JSON data consistency, validity, and interoperability at scale.  
```

Comment:
Could you put line breaks between the paragraphs. Just makes it easier to read.

Reply:
done